### PR TITLE
Ready for stakeholder

### PR DESCRIPTION
**Pull request by TomHessburg**
See my last PR into QA for notes.

Everything checks out to me for now, If at least two of y'all could give this another once over I would appreciate it. Make comments in your reviews and take some time really thinking about the code. Good reviews save a lot of time later :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,155 @@
...
+```
+
+### OpenTok
+```
+We used this to faciliate real-time video conferencing...
+
+Documentation - 
+```
+
+### WhatsApp
```
Comment (micahjones13) [inline README.md]: Not using whatsapp.

File: src/actions/auth.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    };
+  } else {
+    return dispatch => {
+      dispatch({
+        type: GET_UESR_INFO_FAIL,
+        message: "Unable to Get User Data"
+      });
+    };
+  }
+};
```
Comment (micahjones13) [inline src/actions/auth.js]: We honestly could get rid of this whole file. For any current user info, we are using state.firebase.profile or state.firebase.auth

File: src/actions/meetings.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ export const createMeeting = meeting => dispatch => {
   // meeting = meeting data?
   dispatch({ type: MEETING_CREATE });
   return axios
-    .post("/api/meetings", user) //! endpoint unknonwn
+    .post("/api/meetings", meeting) //! endpoint unknonwn
     .then(res => {
       dispatch({ type: MEETING_CREATE_SUCCESS, payload: res.data }); //! Not sure what to return yet
     })
```
Comment (micahjones13) [inline src/actions/meetings.js]: This action is not being used.

File: src/components/views/AwaitingApproval.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ 
+    render() {
+        return (
+            <MDBContainer fluid className = 'd-flex flex-column align-items-center' >
+                <h2 className = 'p-2'>Your application has been submitted!</h2>
+                <h4>Here's what happens next: </h4> 
+                <ul className = 'p-3'>
+                    <li className = 'p-1'>Your application has been sent to an administrator for approval.</li>
+                    <li className = 'p-1'>Once your application is accepted, you can start exploring the app!</li>
+                    <li className = 'p-1'>Please contanct CONTACTINFOHERE for further information regarding your application.</li>
```
Comment (micahjones13) [inline src/components/views/AwaitingApproval.js]: Need valid contact info to fill in CONTACTINFOHERE

File: src/components/calendar/MeetingModal.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+import { firestore, auth } from "../../config/fbConfig.js";
+import SearchResults from "./SearchResults";
+
+const MeetingModal = props => {
+  const [meeting, setMeeting] = useState({
+    title: "",
+    start: Date.now()
+    // endDate: ""
+  });
+  const [participants, setParticipants] = useState({});
```
Comment (Full-lifey) [inline src/components/calendar/MeetingModal.js]: Current or added participants in the selected meeting.

File: src/components/views/AwaitingApproval.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    }
+}
+
+
+
+export default AwaitingApproval;
+
+
+/*
+here is that progress bar I was using.
```
Comment (micahjones13) [inline src/components/views/AwaitingApproval.js]: Can def get rid of this comment.

File: src/components/views/MentorProfile.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+// const mapStateToProps = state => {
+//   return {
+//     auth: state.firebase.auth,
+//     userInfo: state.firebase.profile
+//   };
+// };
+
+// export default connect(mapStateToProps)(MentorProfile);
+
+//DELETE SOOOOOON
```
Comment (micahjones13) [inline src/components/views/MentorProfile.js]: This has been deleted in newer versions

File: src/components/calendar/MeetingModal.js [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+  const [showSearchResults, setShowSearchResults] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+
+  function handleStartDateChange(date) {
```
Comment (Full-lifey) [inline src/components/calendar/MeetingModal.js]: Only used by the datepicker. Could be moved to inside date picker

File: src/components/views/ProfileView.js [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+// //* This state.firebase.profile gives us all the profile info automatically stored into redux by firebase. Neat.
+// const mapStateToProps = state => {
+//   return {
+//     userInfo: state.firebase.profile
+//   };
+// };
+
+// export default connect(mapStateToProps)(ProfileView);
+
+// ///NOT BEING USED. DELETE SOON. HAHA.
```
Comment (micahjones13) [inline src/components/views/ProfileView.js]: Deleted, replaced by ViewUserProfile component

File: src/components/calendar/MeetingModal.js [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+  // }
+
+  const submitMeeting = e => {
+    e.preventDefault();
+    let newParticipants = [];
+    if (participants.length) {
+      newParticipants = participants.map(participant => participant.uid);
+    }
+    newParticipants.push(auth.currentUser.uid);
+    let newMeeting = {
```
Comment (Full-lifey) [inline src/components/calendar/MeetingModal.js]: Checks for selected participants and adds their UID. Then pulls current user's UID and adds it to the metting

File: src/reducers/auth.js [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+      return {
+        ...state,
+        authError: action.err.message
+      };
+    case GET_USER_INFO_SUCCESS:
+      console.log('getuserinfo success');
+      return{
+        ...state,
+        user: {
+          aboutMe: action.userInfo.aboutMe,
```
Comment (micahjones13) [inline src/reducers/auth.js]: All this is coming from the userStore action that got replaced by firebase.profile. Probably no need for it.

File: src/components/calendar/MeetingModal.js [hunk 12]
```diff
@@ -0,0 +1,177 @@
...
+          console.log(doc.data());
+          searchArray.push(doc.data());
+        });
+      });
+    setSearchResults(searchArray);
+  };
+
+  const toggleSearchModal = () => {
+    setShowSearchResults(!showSearchResults);
+  };
```
Comment (Full-lifey) [inline src/components/calendar/MeetingModal.js]: Toggles appearance of search modal

File: src/components/calendar/SearchResults.js [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+} from "mdbreact";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: "100%",
+    maxWidth: 360,
+    backgroundColor: theme.palette.background.paper
+  }
+}));
+
```
Comment (Full-lifey) [inline src/components/calendar/SearchResults.js]: Displays checkable results with avatar and name of user

**Pull request merged**